kde-plasma/breeze-gtk/breeze-gtk-6.0.0.ebuild

```diff
@@ -3,6 +3,7 @@
 
 EAPI=8
 
+KFMIN=5.240.0
```

Comment:
If you check build system (and any other 6.0.0 ebuild) you'll notice `5.240.0` is too low a minimum version.